controllers/nodehealthcheck_controller.go

```diff
@@ -539,14 +595,11 @@ func (r *NodeHealthCheckReconciler) remediate(node *v1.Node, nhc *remediationv1a
 	if err != nil {
 		//An unhealthy node exist but remediation couldn't be created because lease wasn't obtained - update unhealthy nodes and requeue
 		if _, isLeaseAlreadyTaken := err.(lease.AlreadyHeldError); isLeaseAlreadyTaken {
-			resources.UpdateStatusNodeUnhealthy(node, nhc)
 			return leaseRequeueIn, nil
 		}
 
 		//Lease is overdue
 		if _, isLeaseOverDue := err.(resources.LeaseOverDueError); isLeaseOverDue {
-			resources.UpdateStatusNodeUnhealthy(node, nhc)
```

Comment:
Why we have stopped updating the status here (and above)?

Reply:
it's already done earlier now, also covering the case we don't start remediation, see `for _, node := range matchingNodes{}` loop in `Reconcile()`